website/docs/reference/architecture/README.md

```diff
@@ -0,0 +1,51 @@
+# Architecture
+
+Appsmith is available via the [cloud](http://app.appsmith.com/user/sign-up) as well as [self-hosted](/getting-started/setup#self-hosted) instances. It comes in [two versions](https://www.appsmith.com/pricing): Community and Business. Both of these are extremely similar in build and architecture and are deployed as a single Docker container with a single volume for storing persistent data.
```

Comment:
@sharat87 - Should we mention all three versions - Community, Business, and Enterprise?

Reply:
And will the intro remain same for all of them.

Reply:
The architecture diagram and most of the details are describing the Community edition. For Business/Enterprise, there's a few additional components. Can we publish this just for Community edition for now? I'm not sure we want to publicly talk about the architecture of a private repository. What say?

Reply:
Yes. Sounds good. I'll update the introduction. CC: @danciaclara